packages/matrix/tests/messages.spec.ts

```diff
@@ -890,7 +890,47 @@ test.describe('Room messages', () => {
     ]);
   });
 
-  test('attaches a card in a coversation multiple times', async ({ page }) => {
+  test('displays error message if message is too large', async ({ page }) => {
+    await login(page, 'user1', 'pass');
+
+    await page.locator('[data-test-message-field]').fill('a'.repeat(65000));
+    await page.locator('[data-test-send-message-btn]').click();
+
+    await expect(page.locator('[data-test-ai-assistant-message]')).toHaveCount(
+      1,
+    );
+    await expect(page.locator('[data-test-card-error]')).toContainText(
+      'Message is too large',
+    );
+    await expect(page.locator('[data-test-ai-bot-retry-button]')).toHaveCount(
+      0,
+    );
+  });
+
+  /*
+    TODO: This matrix test appears to be leaking state. any tests that are 
+    run after it encounter this failure:
+
+    Error: page.goto: net::ERR_CONNECTION_REFUSED at http://localhost:4202/test
+    Call log:
+      - navigating to "http://localhost:4202/test", waiting until "load"
+
+
+      at ../helpers/index.ts:63
+
+      61 |
+      62 | export async function openRoot(page: Page, url = testHost) {
+    > 63 |   await page.goto(url);
+        |              ^
+      64 |   await expect(page.locator('.cards-grid')).toHaveCount(1);
+      65 |   let isOperatorMode = !!(await page.evaluate(() =>
+      66 |     document.querySelector('dialog.operator-mode'),
+
+        at openRoot (/home/runner/work/boxel/boxel/packages/matrix/helpers/index.ts:63:14)
+        at login (/home/runner/work/boxel/boxel/packages/matrix/helpers/index.ts:232:11)
+        at /home/runner/work/boxel/boxel/packages/matrix/tests/messages.spec.ts:1013:16
+  */
+  test('attaches a card in a conversation multiple times', async ({ page }) => {
```

Comment:
this PR has uncovered some matrix test state leakage.

Reply:
Ah, found the leakage--this test actually results in the realm server crashing--it stops serving!! This test actually mutates the index, which is very undersirable (@lukemelia mentioned something like this during standup today). but in addition to that it also seem to cause some integrity constraints in our index

```
Error executing SQL duplicate key value violates unique constraint "working_queues_pkey" (undefined):
INSERT INTO queues ( queue_name , category , status ) VALUES ( $1 , $2 , $3 ) with bindings: ["default","incremental-index:http://localhost:4202/test/","idle"] error: duplicate key value violates unique constraint "working_queues_pkey"
    at /home/hassan/codez/boxel/node_modules/.pnpm/pg@8.11.5/node_modules/pg/lib/client.js:526:17
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PgAdapter.execute (/home/hassan/codez/boxel/packages/realm-server/pg-adapter.ts:65:22)
    at async query (/home/hassan/codez/boxel/packages/runtime-common/expression.ts:281:10)
    at async PgQueue.query (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:134:12)
    at async PgQueue.publish (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:220:9)
    at async RealmIndexUpdater.update (/home/hassan/codez/boxel/packages/runtime-common/realm-index-updater.ts:138:17)
    at async Realm.write (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:447:5)
    at async Realm.patchCard (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:1504:28)
    at async Router.handle (/home/hassan/codez/boxel/packages/runtime-common/router.ts:159:14)
    at async Realm.handle (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:579:12)
    at async VirtualNetwork.handle (/home/hassan/codez/boxel/packages/runtime-common/virtual-network.ts:124:22)
    at async RealmServer.serveFromRealm (/home/hassan/codez/boxel/packages/realm-server/server.ts:123:25)
    at async convertAcceptHeaderMiddleware (/home/hassan/codez/boxel/packages/realm-server/middleware/convert-accept-header-qp.ts:17:3)
    at async /home/hassan/codez/boxel/packages/realm-server/server.ts:71:9
    at async cors (/home/hassan/codez/boxel/node_modules/.pnpm/@koa+cors@4.0.0/node_modules/@koa/cors/index.js:108:16) {
  length: 270,
  severity: 'ERROR',
  code: '23505',
  detail: 'Key (queue_name, category)=(default, incremental-index:http://localhost:4202/test/) already exists.',
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: 'public',
  table: 'queues',
  column: undefined,
  dataType: undefined,
  constraint: 'working_queues_pkey',
  file: 'nbtinsert.c',
  line: '666',
  routine: '_bt_check_unique'
}
error: duplicate key value violates unique constraint "working_queues_pkey"
    at /home/hassan/codez/boxel/node_modules/.pnpm/pg@8.11.5/node_modules/pg/lib/client.js:526:17
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PgAdapter.execute (/home/hassan/codez/boxel/packages/realm-server/pg-adapter.ts:65:22)
    at async query (/home/hassan/codez/boxel/packages/runtime-common/expression.ts:281:10)
    at async PgQueue.query (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:134:12)
    at async PgQueue.publish (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:220:9)
    at async RealmIndexUpdater.update (/home/hassan/codez/boxel/packages/runtime-common/realm-index-updater.ts:138:17)
    at async Realm.write (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:447:5)
    at async Realm.patchCard (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:1504:28)
    at async Router.handle (/home/hassan/codez/boxel/packages/runtime-common/router.ts:159:14)
    at async Realm.handle (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:579:12)
    at async VirtualNetwork.handle (/home/hassan/codez/boxel/packages/runtime-common/virtual-network.ts:124:22)
    at async RealmServer.serveFromRealm (/home/hassan/codez/boxel/packages/realm-server/server.ts:123:25)
    at async convertAcceptHeaderMiddleware (/home/hassan/codez/boxel/packages/realm-server/middleware/convert-accept-header-qp.ts:17:3)
    at async /home/hassan/codez/boxel/packages/realm-server/server.ts:71:9
    at async cors (/home/hassan/codez/boxel/node_modules/.pnpm/@koa+cors@4.0.0/node_modules/@koa/cors/index.js:108:16) {
  length: 270,
  severity: 'ERROR',
  code: '23505',
  detail: 'Key (queue_name, category)=(default, incremental-index:http://localhost:4202/test/) already exists.',
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: 'public',
  table: 'queues',
  column: undefined,
  dataType: undefined,
  constraint: 'working_queues_pkey',
  file: 'nbtinsert.c',
  line: '666',
  routine: '_bt_check_unique'
}
PATCH application/vnd.card+json http://localhost:4202/test/hassan: 500
error: duplicate key value violates unique constraint "working_queues_pkey"
    at /home/hassan/codez/boxel/node_modules/.pnpm/pg@8.11.5/node_modules/pg/lib/client.js:526:17
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PgAdapter.execute (/home/hassan/codez/boxel/packages/realm-server/pg-adapter.ts:65:22)
    at async query (/home/hassan/codez/boxel/packages/runtime-common/expression.ts:281:10)
    at async PgQueue.query (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:134:12)
    at async PgQueue.publish (/home/hassan/codez/boxel/packages/realm-server/pg-queue.ts:220:9)
    at async RealmIndexUpdater.update (/home/hassan/codez/boxel/packages/runtime-common/realm-index-updater.ts:138:17)
    at async Realm.write (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:447:5)
    at async Realm.patchCard (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:1504:28)
    at async Router.handle (/home/hassan/codez/boxel/packages/runtime-common/router.ts:159:14)
    at async Realm.handle (/home/hassan/codez/boxel/packages/runtime-common/realm.ts:579:12)
    at async VirtualNetwork.handle (/home/hassan/codez/boxel/packages/runtime-common/virtual-network.ts:124:22)
    at async RealmServer.serveFromRealm (/home/hassan/codez/boxel/packages/realm-server/server.ts:123:25)
    at async convertAcceptHeaderMiddleware (/home/hassan/codez/boxel/packages/realm-server/middleware/convert-accept-header-qp.ts:17:3)
    at async /home/hassan/codez/boxel/packages/realm-server/server.ts:71:9
    at async cors (/home/hassan/codez/boxel/node_modules/.pnpm/@koa+cors@4.0.0/node_modules/@koa/cors/index.js:108:16) {
  length: 270,
  severity: 'ERROR',
  code: '23505',
  detail: 'Key (queue_name, category)=(default, incremental-index:http://localhost:4202/test/) already exists.',
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: 'public',
  table: 'queues',
  column: undefined,
  dataType: undefined,
  constraint: 'working_queues_pkey',
  file: 'nbtinsert.c',
  line: '666',
  routine: '_bt_check_unique'
}
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed.
```